doc/manual/manual.tex

```diff
@@ -7891,39 +7893,21 @@ \subsubsection{Inner core convection}
-and thermal diffusion dominates the heat transport. 
-For higher values of $Ra > Ra_c$, the style of convection in the inner core depends on $\mathcal{P}$: 
-For high $\mathcal{P}$ ($\gg 29$, see \cite{Deguen2013}), convection is dominated by thermal plumes with the number 
-of plumes scaling with the Rayleigh number; whereas for low values of $\mathcal{P}$, the inner core is in a 
-translation regime, where material freezes at one side and melts at the other side, so that the velocity field 
-is uniform, pointing from the freezing to the melting side. 
+Three main areas can be distinguished: the stable area, the plume convection area and the translation mode of convection area (Figure~\ref{fig:diagramme-regime}). For low Rayleigh numbers (below the critical value $Ra_c$), there is no convection and thermal diffusion dominates the heat transport. However, if the inner core is convectively unstable ($Ra$>$Ra_c$), the convection regime depends mostly on $\mathcal{P}$. For low $\mathcal{P}$ (<29), the convective translation mode dominates, where material freezes at one side and melts at the other side, so that the velocity field is uniform, pointing from the freezing to the melting side. Otherwise, at high $\mathcal{P}$ (>29), convection takes the usual form of thermal convection with shear free boundary and no phase change, that is the one-cell axisymmetric mode at the onset, and chaotic plume convection for larger Rayleigh number. In this case, melting and solidification at the ICB have only a small dynamic effect. At intermediate values of P, the first unstable mode is a linear combination of the high-P convection mode and of the small-P translation mode.
 
-\begin{figure}
-    \centering
-    \begin{tabular}{lcl}
-    \includegraphics[height=0.25\textwidth]{cookbooks/inner_core_convection/convection.png}
-    & \hspace{1cm} &
-    \includegraphics[height=0.25\textwidth]{cookbooks/inner_core_convection/convection_contours.png}
-    \\
-    \includegraphics[height=0.25\textwidth]{cookbooks/inner_core_convection/translation.png}
-    & \hspace{1cm} &
-    \includegraphics[height=0.25\textwidth]{cookbooks/inner_core_convection/translation_contours.png}
-    \\
-    \includegraphics[height=0.25\textwidth]{cookbooks/inner_core_convection/no_convection.png}
-    & \hspace{1cm} &
-    \includegraphics[height=0.25\textwidth]{cookbooks/inner_core_convection/no_convection_contours.png}
-    \end{tabular}
-    \caption{Convection regimes in the inner core for different values of $Ra$ and $\mathcal{P}$.
-             From top to bottom: plume convection, translation, no convection; full 3d models
-             on the left and 2d slices on the right. Temperature is shown as contours, 
-             velocity is shown as arrows.}
-    \label{fig:inner-core-regimes}
+
+\begin{figure}[h]
+\begin{center}
+\includegraphics[height=0.57\textwidth]{cookbooks/inner_core_convection/Diagstab.pdf}
```

Comment:
That figure is really cool! Can you add a sentence that each dot and triangle is one model run done with ASPECT, and the following text will provide more details on how to modify the input file to do those model runs?

Reply:
done